.gitlab-ci.yml

```diff
@@ -277,31 +277,26 @@ variables:
 .if_not_run_all_builds: &if_not_run_all_builds
   if: $CI_COMMIT_BRANCH != "main" && $DEPLOY_AGENT != "true" && $RUN_ALL_BUILDS != "true"
 
-# Rule to trigger test kitchen setup, run, and cleanup.
+# Rule to trigger test setup, run, and cleanup.
```

Comment:
```suggestion
# Rule to trigger installer tests
```
It seems this comment block is only about the new `if_installer_tests` rule, as such this line is probably useless